config/jobs/kubernetes/test-infra/test-infra-canaries.yaml

```diff
@@ -53,3 +53,40 @@ periodics:
     testgrid-dashboards: sig-testing-canaries
     testgrid-tab-name: multiple-container-test
     description: echo from two different containers
+- interval: 1h
+  name: ci-test-infra-larger-multiple-container-test
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  labels:
+    preset-service-account: "true"
```

Comment:
This should not be required. It is rampantly misused for legacy reasons, but should only be required for old undecorated jobs that are still using the deprecated `bootstrap.py` instead of the pod utilities.